scripts/build-prep-1node-cortx-mgw.sh

```diff
@@ -88,10 +88,11 @@ echo '[client.rgw.8000]'
 echo '      ...'
 echo '      motr my endpoint  = inet:tcp:10.0.0.1@5001'
 echo '      motr my fid       = 0x7200000000000001:0x29'
-echo 'Then run:  MDS=0 RGW=1 ../src/vstart.sh -d $mgs'
+echo "Current directory: $PWD"
+echo "Then run:  MDS=0 RGW=1 ../src/vstart.sh -d $mgs"
 echo 'check:     hctl status'
 echo 'm0client should be in a started state'
 echo 'To stop the cluster run: ../src/stop.sh'
 echo 'Then:  hctl shutdown'
-echo 'If local directory is used then after reboot of the node,'
-echo 'run: mount "/dev/$available_device" $CORTX_WDIR'
+echo 'If local directory is not used then after reboot of the node,'
+echo "run: mount "/dev/$available_device" $CORTX_WDIR"
```

Comment:
remove inner pair of double quotes to overcome Codacy warning.

Reply:
Fixed it.